mega-zep-connector/src/main/java/connector/filter/CORSResponseFilter.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019 by gepardec GmbH.
+ * Autor: Philipp Wurm <philipp.wurm@gepardec.com>
+ * Create date: 02.05.19 19:16
+ */
+
+package connector.filter;
+
+import org.apache.http.HttpStatus;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+import java.io.IOException;
+import java.util.Collections;
+
+@Provider
+public class CORSResponseFilter implements ContainerResponseFilter {
+    @Override
+    public void filter(ContainerRequestContext requestContext,
+                       ContainerResponseContext responseContext) throws IOException {
+        responseContext.getHeaders().add("Access-Control-Allow-Origin", "*");
```

Comment:
Redundancy: those headers are defined in mega-zep-connector/src/main/java/connector/common/GlobalTypes.java.